### ISSUE TITLE
app:transformClassesWithDexForDebug Error

### ISSUE THREAD

**Issue by cherryyingzizhang** [andruhon/android5xlsx#7]
Hello,

Thank you for making this repo. 

However, I have one gradle build issue when trying to run my app on a device:

> Error:Execution failed for task ':app:transformClassesWithDexForDebug'.
> 
> > com.android.ide.common.process.ProcessException: org.gradle.process.internal.ExecException: Process 'command '/Library/Java/JavaVirtualMachines/jdk1.7.0_67.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1

Here is my app's gradle file:

```
apply plugin: 'com.android.application'

android {
    compileSdkVersion 23
    buildToolsVersion "23.0.2"

    defaultConfig {
        applicationId 'com.example.myappname'
        minSdkVersion 21
        targetSdkVersion 23
        multiDexEnabled true
    }

    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
        }
    }

    productFlavors {
    }

    packagingOptions {
        exclude 'META-INF/DEPENDENCIES'
        exclude 'META-INF/NOTICE'
        exclude 'META-INF/NOTICE.txt'
        exclude 'META-INF/LICENSE'
        exclude 'META-INF/LICENSE.txt'
    }
}

dependencies {
    //compile 'com.android.support:support-v4:19.1.0'
    compile project(':MyScript_ATK-Core-android-component')
    compile project(':MyScript_ATK-IMCommon-android-component')
    compile project(':MyScript_ATK-SingleCharWidget-android-widget')
    compile project(':MyScript_ATK-Text-android-component')
    compile 'com.github.amlcurran.showcaseview:library:5.4.3'
    compile 'com.github.douglasjunior:android-simple-tooltip:0.1.0'
    compile 'com.squareup.picasso:picasso:2.5.2'
    compile 'org.apache.poi:poi:3.15-beta2'
    compile 'org.apache.poi:poi-ooxml:3.15-beta2'
    compile 'com.android.support:multidex:1.0.1'
}
```

And in my project gradle file, I have gradle version 1.5.0, like in your example:

`classpath 'com.android.tools.build:gradle:1.5.0'`

If you could help me with this, I really appreciate it!

**Comment by andruhon** [comment]
Hi! Could you please also provide your proguard config?

**Comment by cherryyingzizhang** [comment]
Hello, thanks for your reply.

According to the answers I see on this [stackoverflow](http://stackoverflow.com/questions/23149417/proguard-project-txt-not-found-in-android-studio) question, It seems that because I imported this old Eclipse project into Android Studio, proguard-rules.txt did not actually exist in my project directory. Instead I now replaced it with a default-made proguard-rules.pro file.

I also made an example app doing the same steps over again (with the default-generated proguard-rules.pro), and fortunately I got it to work this time. Seems like the issue is resolved.
